pages/index.mdx

```diff
@@ -47,12 +48,13 @@ export const BannerWrapper = styled.div`
 
 export const ContentContainer = styled.div`
   padding: 60px;
+  > h3 {
+    margin: unset;
+    margin: 60px 0px 17px 0px;
+  }
 `;
 
-export const CardGrid = styled.div`
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
-  column-gap: 30px;
+export const CardGridWrapper = styled.div`
```

Comment:
This will make the cards stack on top of each other like the Guides/Concepts cards.